Orange/widgets/utils/domaineditor.py

```diff
@@ -265,8 +265,11 @@ def get_domain(self, domain, data):
             col_data = self._get_column(data, orig_var, orig_plc)
             is_sparse = sp.issparse(col_data)
 
-            cont_ints = type(orig_var) == ContinuousVariable and \
-                        all(x.is_integer() for x in self._iter_vals(col_data) if not np.isnan(x))
+            def cont_ints():
+                if type(orig_var) == ContinuousVariable:
+                    data = col_data.data[np.isfinite(col_data.data)]
+                    return (data == data.astype(np.int16)).all()
```

Comment:
Maybe `np.int64`? As I understand, these number-like strings can be arbitrarily large, so 64-bit ints are needed to cover anything that can be stored in `float64`. (If the file contains a number-like string that is larger than that, it is already lost before arriving here.)